docs/modules/ROOT/pages/cops.adoc

```diff
@@ -39,6 +39,7 @@ based on the https://rails.rubystyle.guide/[Rails Style Guide].
 * xref:cops_rails.adoc#railsdefaultscope[Rails/DefaultScope]
 * xref:cops_rails.adoc#railsdelegate[Rails/Delegate]
 * xref:cops_rails.adoc#railsdelegateallowblank[Rails/DelegateAllowBlank]
+* xref:cops_rails.adoc#railsduplicateignoredcolumns[Rails/DuplicateIgnoredColumns]
```

Comment:
Please revert doc changes. Those will be generated with the new version release.